commerce/bookstore/python/test.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2022 Vaticle
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from unittest import mock, TestCase
+import load_data
+import requests
+import config
+from typedb.client import TypeDB, SessionType, TransactionType
+
+class LoadDataTests(TestCase):
```

Comment:
See discussion in Discord.

Reply:
Fixed comments in https://github.com/vaticle/typedb-examples/pull/143/commits/1d54bc4e40f9e64ddf3ca469797d0930459b651b
Also, I added the numbers. Otherwise, it's hard to decipher this comment. Too many non-numbered elements in the list